agent/cvms/api/grpc/client.go

```diff
@@ -58,6 +100,43 @@ func (client *CVMSClient) Process(ctx context.Context, cancel context.CancelFunc
 	return eg.Wait()
 }
 
+func (client *CVMSClient) loadPendingMessages() error {
```

Comment:
I like the naming where `CVMS` is capitalize (S also) - but can we check if this is aligned in all code? i think  I saw one PR from @WashingtonKK where it was `CVMs`